### PR TITLE
fix: subcommand completion for each extension

### PR DESCRIPTION
**Pull request by tamago324**
Fixed subcommand completion for each extension.

@glepnir If there is a reason why the subcommands for all extensions are being completed, I would appreciate it if you could let me know.

## Before

https://user-images.githubusercontent.com/16581287/109809572-c85fd280-7c6b-11eb-8398-062ba1827eca.mp4

## After

https://user-images.githubusercontent.com/16581287/109809677-f0e7cc80-7c6b-11eb-80bb-db6359856bb6.mp4

### CODE REVIEW

**Review by Conni2461 — APPROVED** [review]
Finally :heart: Thanks :)

LGTM should still wait for @glepnir approval tho

**Comment by glepnir** [comment]
LGTM

**Comment by tamago324** [comment]
Thanks!

**Pull request merged**
